config/watcher/operator_v1alpha1_watcher.yaml

```diff
@@ -13,7 +13,8 @@ spec:
     name: event-service
     port: 8082
     namespace: kcp-system
-
+  gateway:
+    namespacedName: default/kyma-gateway
```

Comment:
I think @ruanxin was changing this, maybe sync there on what is now the goto, namespacedName or label?

Reply:
Yes, we will use `LabelSelector`, `namespacedName` will be removed.